hr_employee_operating_unit/security/security.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+<data noupdate="0">
```

Comment:
data it's not necessary anymore. Use <odoo noupdate="0">

Reply:
Sorry. I meant <odoo noupdate="0"> but it does not matter because that it's the same as <odoo>